### DISCUSSION TITLE
Tennis markets

### DISCUSSION THREAD

**Discussion by thomasjones11** [cloudbet/docs#30] [Q&A]
Can I ask if it would be possible to add the total games and game handicap markets for tennis events? Also could there be a route added to reload play money for testing purposes. Asking in the live chat in the website but they would only reload to 100.

**Comment by thomasjones11** [comment]
The total sets market doesn't return for me right now for any tennis match but it was a while ago. I dunno if that is a bug?

**Comment by kgravenreuth** [comment]
> Can I ask if it would be possible to add the total games and game handicap markets for tennis events? Also could there be a route added to reload play money for testing purposes. Asking in the live chat in the website but they would only reload to 100.

Hey @thomasjones11 , we'll review these markets and get back to you here soon.

> The total sets market doesn't return for me right now for any tennis match but it was a while ago. I dunno if that is a bug?

The total sets market could become available or be removed for live tennis matches depending on the current match situation. For example, if a 3 set tennis match is already in the 3rd set, then the total sets market doesn't make sense anymore. 

If you see this issue again unexpectedly, could you please let us know about which events you were checking, as well as the match state (such as live or upcoming event, current set)? Thanks.

**Reply by thomasjones11** [reply]
Thanks. The issue with the total sets was happening for matches before they start. It's happening again now for FO mens singles but the markets are not on the website either right now so I suppose it is not a bug maybe the market is just offline for now.
